lib/chatBot.js

```diff
@@ -971,7 +1004,7 @@ ChatBot.prototype.makeAnnouncement = function(target, head, body, source) {
 	var post_req = http.request(post_options, function(res) {
 		res.setEncoding("utf8");
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['chunk' is defined but never used.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/946709221/issues/source?bid=1980036&fileBranchId=2848073#l1006)